src/Store/StoreSections.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
```

Comment:
Hi! You can also use import to grab module members, which makes the code a little neater, like this:
```
import React, { Component, PropTypes } from "react";
...
export default class StoreSections extends Component {
  static propTypes = {
    title: PropTypes.string
  }
}
```